### DISCUSSION TITLE
Notifications Support?

### DISCUSSION THREAD

**Discussion by peterjohnhunt** [johnlindquist/kit#165] [Q&A]
I know the [project-name](https://scriptkit.app/scripts/johnlindquist/project-name.js) script mentions showing a notification. I wasn't able to get that script to work specifically, but is [notifications](https://www.npmjs.com/package/node-notifier) something you're thinking about including in the available built in functions?

Could open some cool interactive scripts to run in background or be triggered by other things behind the scenes with quick native style notifications.

**Comment by johnlindquist** [comment]
This works:

```js
let { notify } = await kit("desktop")

notify("Some title", "some subtitle")
```

But I'll have to look into that npm package. Having callbacks sounds interesting...

Thanks for the link!

**Reply by peterjohnhunt** [reply]
ohhh, i shoulda checked under desktop! thanks for that!
